### ISSUE TITLE
Can't sync project with gradle files

### ISSUE THREAD

**Issue by bogomazov** [wix/react-native-navigation#5374]
### Issue Description

Once I followed the steps for the installation on Android and try to sync gradle files (to have com.reactnativenavigation package available to use in MainActivity and MainApplication) I get the following error:

```
ERROR: The module 'react-native-navigation' is an Android project without build variants, and cannot be built.
Please fix the module's configuration in the build.gradle file and sync the project again. 
```
### Steps to Reproduce / Code Snippets / Screenshots

Start a new project with RN 0.60 and follow the steps in the instructions for Android.

---
### Environment
* React Native Navigation version: 2.26.5
* React Native version: 0.60.4
* Platform(s) (iOS, Android, or both?): Android
* Device info (Simulator/Device? OS version? Debug/Release?): Android Studio Sync Gradle Files

**Comment by PerBear** [comment]
I have the same problem. After looking through the documentation it looks like it could use some updating

**Comment by jinshin1013** [comment]
Guys if you wish to use this library with react-native@0.60.*, you need to use react-native-navigation@3.0.0-alpha version 

And since it's still in alpha you can access the new updated installation guide manually in `/doc` folder in v3 branch.

**Comment by Abramovick** [comment]
@jinshin1013 I've switched to v3 branch but i still see the old docs for some reason https://github.com/wix/react-native-navigation/blob/v3/docs/docs/Installing.md...

**Comment by jinshin1013** [comment]
V3 doc should now be in the master branch as it got merged.

**Comment by Abramovick** [comment]
I've followed the [installation docs](https://github.com/wix/react-native-navigation/blob/master/docs/docs/Installing.md) on master and it doesn't work for me. 
I'm using react-native-navigation@3.0.0-alpha.2

Can you please paste for me what you currently have working on `MainApplication.Java` and `MainActivity`

**Comment by Abramovick** [comment]
Managed to get it working thanks to @sijad.
I'll just drop a link here for anyone using react `> 60.*.*` and wants to use react-native-navigation, here's a great [working example](https://github.com/wix/react-native-navigation/issues/5296#issuecomment-513204708)

**Comment by crocodile** [comment]
I had the same error _"The module 'react-native-navigation' is an Android project without build variants, and cannot be built."_ but then I realised I accidentally filtered out all variants even the latest one that I needed when I was following instructions at [step 5.2](https://wix.github.io/react-native-navigation/#/docs/Installing). It seems to be working now. I published my project configuration with React Native 0.61.2 and react-native-navigation 3.2.0 to a [GitHub repo](https://github.com/crocodile/react-native-navigation-base).
